### PR TITLE
Meta tags, title, OG, Cache Control in .htaccess

### PR DESCRIPTION
**Pull request by tradzik**

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<IfModule mod_headers.c>
+<FilesMatch "\.(jpg|jpeg|png|gif|swf|JPG)$">
+Header set Cache-Control "max-age=4838400, public"
+</FilesMatch>
+<FilesMatch "\.(css|js)$">
+Header set Cache-Control "max-age=4838400, private"
+</FilesMatch>
+</IfModule>
```
Comment (samdroid-apps) [inline .htaccess]: I added these rules in nginx, so I don't think we need this file.

Actually, does nginx even read the .htaccess files?

**Comment by samdroid-apps** [comment]
Looks good!  Just could probably drop the htaccess.

**Comment by davelab6** [comment]
I agree, looks good

File: .htaccess (hunk 1 above)
Comment (davelab6) [inline .htaccess]: There's no harm in having that file included in the repo, right?

**Comment by davelab6** [comment]
I suggest this can be merged.

**Comment by davelab6** [comment]
Replaced with #40

**Pull request closed without merging**